src/utils/featheryClient/integrationClient.ts

```diff
@@ -378,8 +378,27 @@ export default class IntegrationClient {
       if (!action.html_url_field)
         throw Error('No field selected for storing HTML URL returned by Quik!');
 
-      payload.html_url_field = action.html_url_field;
-      payload.html_url_field_type = action.html_url_field_type;
+      Object.assign(payload, {
+        html_url_field: action.html_url_field,
```

Comment:
honestly feels like we can just pass the entire action into the payload and backend can parse what itneeds